cmd/analyze.go

```diff
@@ -51,21 +50,20 @@ var analyzeCmd = &cobra.Command{
 
 func checkAnalyzeArgNum(args []string) error {
 	if len(args) != 1 {
-		return errors.New("'analyze' requires one image as an argument: container analyze [image]")
+		return fmt.Errorf("'analyze' requires one image as an argument: container-diff analyze [image]")
```

Comment:
errors.New is fine here, it's basically the same as fmt.Errorf except fmt does formatting. You're not adding any arguments so we should stick to errors.New

Reply:
done